src/content/datachannel/datatransfer/js/main.js

```diff
@@ -24,6 +24,13 @@ var bytesToSend = 0;
 
 sendButton.onclick = createConnection;
 
+// Prevent data sent to be set to 0.
+megsToSend.addEventListener('change', function(e) {
+  if (this.value === 0) {
+    megsToSend.value = 1;
```

Comment:
Valid point, I've changed it to be clearer and also it disables/enables the button, WDYT?